Dependencies/AndroidExtensions/Source/Globals.cpp

```diff
@@ -55,10 +55,10 @@ namespace android::global
         RequestPermissionsResultEvent g_requestPermissionsResultEvent{};
     }
 
-    void Initialize(JavaVM* javaVM, jobject appContext)
+    void Initialize(JavaVM* javaVM, jobject context)
     {
         g_javaVM = javaVM;
-        g_appContext = appContext;
+        g_appContext = GetEnvForCurrentThread()->NewGlobalRef(android::content::Context{context}.getApplicationContext());
```

Comment:
Need to release this?

Reply:
Nowhere. It's not obvious to me where we would do this, and this is one jobject referencing an object that already will never get deleted, so I think it is fine.